internal/controller/controllers/error_utils.go

```diff
@@ -30,3 +30,14 @@ func isClientError(err error) bool {
 		return false
 	}
 }
+
+func hasStatusCode(err error, code int32) bool {
```

Comment:
this looks like [IsHTTPError](https://github.com/openshift/assisted-service/pull/939/files#diff-07a00de246469cec43cc844a62470b44628315b1d6ffaed0d5d3e5280e33fa7dR36) from pr https://github.com/openshift/assisted-service/pull/939, we should consolidate those, once one of the PRs gets merged (no action to take for now).

Reply:
https://github.com/openshift/assisted-service/pull/939 got merged, so please rebase and reuse I[sHTTPError](https://github.com/openshift/assisted-service/blob/fc5b403fa34b83422cbe28f6a882e73c830dcfef/internal/controller/controllers/error_utils.go#L36)